neurodamus/connection_manager.py

```diff
@@ -330,7 +330,8 @@ def _init_conn_population(self, src_pop_name):
         )
         logging.info("Loading connections to population: %s", cur_pop)
 
-    def _compute_pop_ids(self, src_pop_name, dst_pop_name):
+    @staticmethod
```

Comment:
I do not see the reason of this change. Static method means that you can use the method without instantiating the class. The method is `_*` meaning that it should be used only inside the class (internal use). For what case is this relevant apart from the fact that we do not need to pass self and is cleaner?

Reply:
It's the correct way to do things; otherwise it's not clear why the self is there.
Whether or not it's a "private" method doesn't matter - they are orthogonal concepts; it's more that it clearly signals the intent.

Reply:
mmm ok. All the conversations about this point can be resolved imo